modules/manage/partials/monitor-connectors.adoc

```diff
@@ -0,0 +1,321 @@
+--values prometheus-monitoring.yaml --reuse-values
+```
+
+--set::
++
+[,bash,lines=4-6]
+----
+helm upgrade --install redpanda redpanda/redpanda \
+  --namespace <namespace> \
+  --create-namespace \
+  --set connectors.enabled=true \
+  --set connectors.monitoring.enabled=true \
+  --set connectors.monitoring.scrapeInterval="30s"
+----
+
+====
+--
+======
+
+. Wait until all Pods are running:
++
+[,bash]
+----
+kubectl -n <namespace> rollout status statefulset redpanda --watch
+----
+
+. Ensure that the PodMonitor was deployed:
++
+[,bash]
+----
+kubectl get podmonitor --namespace <namespace>
+----
+
+. Ensure that you've https://prometheus-operator.dev/docs/user-guides/getting-started/#exposing-the-prometheus-service[exposed the Prometheus Service^].
+
+. Expose the Prometheus server to your localhost:
++
+[,bash]
+----
+kubectl port-forward svc/prometheus 9090
+----
+
+. http://localhost:9090/graph[Open Prometheus^], and see that Prometheus is scraping metrics from your endpoints.
+
+endif::[]
+
+ifndef::env-kubernetes[]
+
+NOTE: You can quickly get
+https://github.com/redpanda-data/observability/tree/main/cloud[Prometheus
+and Grafana running locally^], but not for production instances. For
+production instances, deploy Prometheus and Grafana as a standalone or
+managed service, as described below.
+
+To configure and use Prometheus to monitor Redpanda managed connector
+metrics:
+
+. In Redpanda Cloud, go to *Overview > *How to connect* > *Prometheus*.
+Click the Copy icon for *Prometheus YAML* to copy its content into your
+clipboard.
+. Edit the `prometheus.yml` file in the Prometheus root folder to add
+the Redpanda configuration under `scrape_configs`.
++
+[,yaml]
+----
+scrape_configs:
+- job_name: redpandaCloud
+    static_configs:
+    - targets:
+        - ...
+    metrics_path: /api/cloud/prometheus/public_metrics
+    basic_auth:
+    username: prometheus
+    password: ...
+    scheme: https
+----
+
+. Save the configuration file, and restart Prometheus to apply changes.
+. Observe in Prometheus that metrics from Redpanda endpoints are
+scraped.
+
+endif::[]
+
+== Import the Grafana dashboard
+
+You can use https://grafana.com/oss/grafana/[Grafana^] to query,
+visualize, and generate alerts for metrics. Redpanda provides a
+https://github.com/redpanda-data/observability/blob/main/grafana-dashboards/Connectors.json[Grafana
+dashboard for connectors^].
+
+To create and use the Grafana dashboard to gather telemetry for your
+managed connectors, import the connectors dashboard JSON file
+(`Connectors.json`).
+
+== Managed connector metrics
+
+You can monitor the following metrics for your Redpanda managed
+connectors.
+
+=== Connector tasks
+
+Number of tasks for a specific connector, grouped by status:
+* `running` - Tasks that are healthy and running.
+* `paused` - Tasks that were paused by a user request.
+* `failed` - Tasks that failed during execution.
```

Comment:
Not sure why these are not rendering as a list...

Reply:
It's due to the missing space before the list 👍